js/views/components/Moderators.js

```diff
@@ -14,35 +14,28 @@ export default class extends baseVw {
     if (!options.fetchErrorTitle) {
       throw new Error('Please provide error text for the moderator fetch.');
     }
-    if (!options.cardState) {
-      throw new Error('Please provide a default card state.');
-    }
-    if (!options.notSelected) {
-      throw new Error('Please provide a default not selected state.');
-    }
 
     /**
-     * @namespace
-     * @property {string}  apiPath           - Current options are fetchprofiles and moderators.
-     * @property {boolean} async             - Return profiles via websocket.
-     * @property {boolean} useCache          - Use cached data for faster speed.
-     * @property {array}   moderatorIDs      - list of moderators to retrieve. If none get all.
-     * @property {array}   excludeIDs        - list of moderators to not use.
-     * @property {string}  method            - POST or GET
-     * @property {string}  include           - If apiPath is moderator, set to 'profile' or only the
-     *                                         peerIDs of each moderator are returned.
-     * @property {boolean} purchase          - If this is used in a purchase, pass this to the
-     *                                         child moderator card views.
-     * @property {boolean} singleSelect      - Allow only one moderator to be selected at a time.
-     * @property {boolean} selectFirst       - Pre-select the first moderator.
-     * @property {boolean} radioStyle        - Show the moderator cards with radio buttons.
-     * @property {boolean} showInvalid       - Show invalid moderator cards.
-     * @property {boolean} controlsOnInvalid - Show controls on invalid cards so they can be removed
-     *                                         or otherwise acted on.
-     * @property {string}  wrapperClasses    - Add classes to the card container.
-     * @property {string} fetchErrorTitle    - A title for the fetch error.
-     * @property {string} cardState          - The initial state for cards that are created.
-     * @property {string} notSelected        - Which not selected state to use on the mod cards.
+     * @param {string}  apiPath           - Current options are fetchprofiles and moderators.
```

Comment:
The js documentation here should go above the constructor signature. Also, the params that are properties of the `options`, need to be prefaced as such (singleSelect ==> options. singleSelect), since you could also document arguments passed into the constructor which are not part of the options object.

An example of this is here:
https://stackoverflow.com/questions/10683430/jsdoc-how-do-i-document-the-options-object-literal-for-a-parent-class/14820610#14820610

It's also pretty much the style we're using in the rest of the code base.